TerrorConsole/Assets/_Main/Scripts/Objects/Switchs/SwitchObject.cs

```diff
@@ -0,0 +1,41 @@
+using UnityEngine;
+
+namespace TerrorConsole
+{
+    public class SwitchObject : MonoBehaviour
+    {
+        [SerializeField] private MonoBehaviour _switchInteractuableObject;
+        protected ISwitchInteractuableSource _interactuableObject;
```

Comment:
Not needed if there is only 1 class that will implement the interface, if there will be more tell me, maybe we need the interface.